### DISCUSSION TITLE
[Bug] log in error ""Passphrase is Incorrect""

### DISCUSSION THREAD

**Discussion by madsold202** [Chia-Network/chia-blockchain#9431] [Support]
### What happened?

Dear Sir/Madam

Good day;

After i downloaded the new version of Chia Software 1.2.11, unfourtanutly i can not Log in to my Chia Account i got error "Passphrase is Incorrect", but im sure i did true, because i logged in before with Version 1.2.10.

how i can resolve the problem.

Best regards
Mahdi

### Version

1.211

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by loppefaaret** [comment]
If you are unable to remember the Password protection - you can close down Chia, and move the file for the keyring out:
`~\.chia_keys\keyring.yaml`
after moving this out, the password would be gone, and it should ask you to create a new, after migrating keys into it
